### PR TITLE
Missing commits from modin branch

### PR DESCRIPTION
**Pull request by Garra1980**
I have only 
68: Error while processing /localdisk/izamyati/omniscidb_new/build/Tests/../../Tests/Udf/udf_sample.cpp.
68: unknown file: Failure
68: C++ exception with description "Unable to create AST file for udf compilation (error code 1)" thrown in auxiliary test code (environments or event listeners).
62/62 Test #68: UdfTest ............................***Failed    0.19 sec

### CODE REVIEW

**Comment by ienkovich** [comment]
I guess you are using conda and there is a fix for that in scripts/conda/build.sh which might help you with testing:
```
# Omnisci UDF support uses CLangTool for parsing Load-time UDF C++
# code to AST. If the C++ code uses C++ std headers, we need to
# specify the locations of include directories:
. ${RECIPE_DIR}/get_cxx_include_path.sh
export CPLUS_INCLUDE_PATH=$(get_cxx_include_path)
```

File: Shared/InlineNullValues.h [hunk 1]
```diff
@@ -366,4 +366,16 @@ CONSTEXPR DEVICE inline void set_null(T& value) {
   *(TT*)(&value) = serialized_null_value<T, array>();
 }
 
+template <typename V,
```
Comment (ienkovich) [inline Shared/InlineNullValues.h:369]: This actually caused #250. You just don't see it in your testing because of another error.

Comment (Garra1980) [reply Shared/InlineNullValues.h:369]: I thought this was a compile-time error - I don't see it while building (I'm building tests as well)

**Comment by Garra1980** [comment]
> I guess you are using conda and there is a fix for that in scripts/conda/build.sh which might help you with testing:
> 
> ```
> # Omnisci UDF support uses CLangTool for parsing Load-time UDF C++
> # code to AST. If the C++ code uses C++ std headers, we need to
> # specify the locations of include directories:
> . ${RECIPE_DIR}/get_cxx_include_path.sh
> export CPLUS_INCLUDE_PATH=$(get_cxx_include_path)
> ```

I have these lines in build.sh...

**Comment by ienkovich** [comment]
> I have these lines in build.sh...

Did you try to run tests defining a similar macro?

File: Shared/InlineNullValues.h (hunk 1 above)
Comment (ienkovich) [reply Shared/InlineNullValues.h:369]: No, it is a runtime error in UDF tests in an attempt to compile a file with a sample UDF function (this file includes `Shared/InlineNullValues.h`).

**Comment by Garra1980** [comment]
> > I have these lines in build.sh...
> 
> Did you try to run tests defining a similar macro?

Yeah, manual run is green for UDFTest, no runtime errors

**Review by ienkovich — APPROVED** [review]

**Pull request merged**